### PR TITLE
Alert and button

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (415)
Screenshot (416)
Screenshot (417)

Screenshot (418)

I made alert and button component. I still have to change px to rems and add the carbonn code.

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+.alert-head{
+    margin: -5px;
+}
+.alert-topic{
+    margin-top: 30px;
+}
+.alert-content{
+    padding: 20px;
+    font-size: 19px;
```
Comment (Hamdrive) [inline components/alert/alert.css]: You should try to use CSS variables with defined sizes (for rems)
allows for alot of reusability

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    color: var(--primary-color);
+}
+.alert-container{
+    display: flex;
+    flex-basis: 400px;
+    flex-direction: column;
+    gap: 30px;
+    padding: 40px;
+    margin-top: 30px;
+    background-color: #dbe0e7;
```
Comment (Hamdrive) [inline components/alert/alert.css]: Add to `:root` variables, similarly like you have for other colors/bg colors

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    font-size: 19px;
+    color: var(--primary-color);
+}
+.btn-container{
+    display: flex;
+    flex-wrap: wrap;
+    flex-basis: 400px;
+    gap: 30px;
+    justify-content: center;
+    align-items: center;
```
Comment (Hamdrive) [inline components/button/button.css:32]: Can use helper classes with predefined properties of flex

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    border: none;
+    border-radius: 5px;
+    font-family: 'Montserrat', sans-serif ;
+}
+.primary-btn{
+    color: whitesmoke;
+    background-color: var(--primary-color);
+}
+.primary-btn:hover{
+    background-color: #325499;
```
Comment (Hamdrive) [inline components/button/button.css:55]: Use CSS variables like you've done in this same file

**Review by Hamdrive — COMMENTED** [review]
Good work done
Suggested some changes which will make building the overall project easier and cleaner

File: components/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    border-radius: 5px;
+}
+.alert-box, .alert-border{
+    height: 50px;
+    max-width: 900px;
+    border-radius: 5px;
+    /* color: whitesmoke; */
```
Comment (junaidshaikh-js) [inline components/alert/alert.css]: code comments should be removed.

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good.

**Pull request merged**